tests/common/templates/pfc_storm_eos.j2

```diff
@@ -1,9 +1,9 @@
 bash
 cd /mnt/flash
 {% if (pfc_asym  is defined) and (pfc_asym == True) %}
-{% if pfc_storm_defer_time is defined %} sleep {{pfc_storm_defer_time}} &&{% endif %} sudo python {{pfc_gen_file}} -p {{pfc_queue_index}} -t 65535 -n {{pfc_frames_number}} -i {{pfc_fanout_interface | replace("Ethernet", "et") | replace("/", "_")}} &
+{% if pfc_storm_defer_time is defined %} sleep {{pfc_storm_defer_time}} &&{% endif %} sudo chrt 98 python {{pfc_gen_file}} -p {{pfc_queue_index}} -t 65535 -n {{pfc_frames_number}} -i {{pfc_fanout_interface | replace("Ethernet", "et") | replace("/", "_")}} > /dev/null 2>&1 &
```

Comment:
Could you please share the summary of the test result if available? 
1: what's the original failure rate without the priority set to 98?
2: Has the issue been resolved with this fix?
3: how does the failure rate compare with other priority levels? is it necessary to set 98 ?
thanks